examples/zephyr/lightdb/get/socs/esp32_procpu.conf

```diff
@@ -1,5 +1,5 @@
 CONFIG_WIFI=y
-CONFIG_HEAP_MEM_POOL_SIZE=49152
+CONFIG_HEAP_MEM_POOL_SIZE=8192
```

Comment:
We can remove this change. This example should run now with the change to disable `CONFIG_ESP_HEAP_RUNTIME`